examples/with-arc/src/index.ts

```diff
@@ -0,0 +1,198 @@
+import * as path from "path";
+import * as dotenv from "dotenv";
+import prompts from "prompts";
+import { ethers } from "ethers";
+
+// Load environment variables from `.env.local`
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
+
+import {
+  getSignedTransactionFromActivity,
+  TurnkeyActivityConsensusNeededError,
+  TERMINAL_ACTIVITY_STATUSES,
+  TActivity,
+} from "@turnkey/http";
+import { TurnkeySigner } from "@turnkey/ethers";
+import { Turnkey as TurnkeyServerSDK } from "@turnkey/sdk-server";
+import { createNewWallet } from "./createNewWallet";
+import { print } from "./util";
+
+async function main() {
+  if (!process.env.SIGN_WITH) {
+    // If you don't specify a `SIGN_WITH`, we'll create a new wallet for you via calling the Turnkey API.
+    await createNewWallet();
+    return;
+  }
+
+  const turnkeyClient = new TurnkeyServerSDK({
+    apiBaseUrl: process.env.BASE_URL!,
+    apiPrivateKey: process.env.API_PRIVATE_KEY!,
+    apiPublicKey: process.env.API_PUBLIC_KEY!,
+    defaultOrganizationId: process.env.ORGANIZATION_ID!,
+    // The following config is useful in contexts where an activity requires consensus.
+    // By default, if the activity is not initially successful, it will poll a maximum
+    // of 3 times with an interval of 1000 milliseconds. Otherwise, use the values below.
+    //
+    // -----
+    //
+    // activityPoller: {
+    //   intervalMs: 5_000,
+    //   numRetries: 10,
+    // },
+  });
+
+  // Initialize a Turnkey Signer
+  const turnkeySigner = new TurnkeySigner({
+    client: turnkeyClient.apiClient(),
+    organizationId: process.env.ORGANIZATION_ID!,
+    signWith: process.env.SIGN_WITH!,
+  });
+
+  const network = "arc testnet";
+  const provider = new ethers.JsonRpcProvider(
+    `https://rpc.testnet.arc.network/`,
+  );
+  const connectedSigner = turnkeySigner.connect(provider);
+
+  const chainId = (await connectedSigner.provider?.getNetwork())?.chainId ?? 0;
+  const address = await connectedSigner.getAddress();
+  const transactionCount =
+    await connectedSigner.provider?.getTransactionCount(address);
+  let balance = (await connectedSigner.provider?.getBalance(address)) ?? 0;
+
+  print("Network:", `${network} (chain ID ${chainId})`);
+  print("Address:", address);
+  print("Balance:", `${ethers.formatEther(balance)} USDC`);
```

Comment:
ETH uses 18 decimal places; we might want a different parsing mechanism given USDC has 6 decimals

Reply:
in any other context - correct, however Arc puts native USDC to 18 decimals https://docs.arc.network/arc/references/evm-compatibility#erc20-interface